migrations/versions/0cd0a3b251e0_unit_contact_email_non_nullable.py

```diff
@@ -0,0 +1,40 @@
+"""unit_contact_email_non_nullable
+
+Revision ID: 0cd0a3b251e0
+Revises: 3d610b382383
+Create Date: 2024-12-03 10:51:34.143028
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "0cd0a3b251e0"
+down_revision = "3d610b382383"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute(
+        "UPDATE units SET contact_email = 'delivery@scilifelab.se' WHERE contact_email IS NULL"
+    )
```

Comment:
Currently other migrations e.g. `666003748d14_change_active_nullable_and_default.py` are using another method of executing these things, and I don't want to introduce different methods, so my suggestion is to change to the following
```suggestion
    unit_table = sa.sql.table("units", sa.sql.column("contact_email", mysql.TINYINT(display_width=1)))
    op.execute(unit_table.update().where(unit_table.c.contact_email == None).values(contact_email="delivery@scilifelab.se"))
```
If there rest of the team @ScilifelabDataCentre/teamhermes disagrees and thinks we should have SQL instead, then I want to change the other migrations to have this structure too.

Reply:
I can change, it is esentially the same so i dont mind

Reply:
I think it makes sense to be consistent, i.e. to use the `sa` (despite of SQL being easier for me to read because I am not that used to `sa` :) )